README.md

```diff
@@ -1,19 +1,16 @@
-### hey, I'm Molim 👋
-
-const molim = {
+# Hey, I'm Molim 👋
 
+```js
```

Comment:
When you want to type code inside markdown, you type inside three backticks. You can also add a language because some places support syntax highlight (it shows with some color).

More here: https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks